### ISSUE TITLE
Dependency conflict

### ISSUE THREAD

**Issue by tokeefe** [nipreps/mriqc#856]
I'm trying to install mriqc using pipenv (`pipenv install mriqc==0.15.2`). The dependency resolver is showing an unreconcilable dependency conflict. This doesn't seem to affect the overall functionality of mriqc, but I figured that I would report it here

```
There are incompatible versions in the resolved dependencies:
  pybids>=0.10.2 (from mriqc==0.15.2->-r /tmp/pipenvwxet1q7_requirements/pipenv-e976ksyc-constraints.txt (line 2))
  pybids>=0.11.1 (from templateflow==0.6.3->mriqc==0.15.2->-r /tmp/pipenvwxet1q7_requirements/pipenv-e976ksyc-constraints.txt (line 2))
  pybids~=0.10.2 (from niworkflows==1.2.7->mriqc==0.15.2->-r /tmp/pipenvwxet1q7_requirements/pipenv-e976ksyc-constraints.txt (line 2))
```

If I'm reading this correctly, templateflow==0.6.3 and niworkflows==1.2.7 cannot agree on a version of pybids.

**Comment by effigies** [comment]
I suspect the short-term fix is setting the niworkflows pin to `~=1.1.12`, rather than `~=1.1` which will take anything up to `2.0.dev0`.

**Comment by HippocampusGirl** [comment]
Are there any downsides to the short-term fix?

**Comment by effigies** [comment]
No. If we haven't done it yet, we should do it ASAP...

**Comment by effigies** [comment]
Looks like it's fixed in 0.16.1: https://github.com/poldracklab/mriqc/blob/0.16.1/setup.cfg#L33

So this is just open to indicate that the long-term fix (updating all dependencies) still needs doing.

**Comment by oesteban** [comment]
With the release of 21.0.0rc0 and rc1 I hope I have addressed all the dependency updates that were necessary. Please reopen if I missed any.
